Sources/AltSwiftUI/Source/UIKitViews/UIKitControls.swift

```diff
@@ -143,9 +146,15 @@ class SwiftUITextField<T>: UITextField, UITextFieldDelegate, UIKitViewHandler {
         let newText = (textField.text as NSString?)?.replacingCharacters(in: range, with: string)
         if let text = newText, textBinding?.wrappedValue != text {
             lastWrittenText = text
-            setBindingText(text)
         }
-        return textBinding?.wrappedValue == newText
+        return true
```

Comment:
I think it's not necessary to keep this method anymore. Logic related to setting up the `lastWrittenText` can be moved over to `textFieldDidChange`